### PR TITLE
[@types/underscore] Add array as key for `property`

### PR DESCRIPTION
**Pull request by lleelleeoo**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: [property](https://underscorejs.org/#property), [propertyOf](https://underscorejs.org/#propertyOf)
- [ ] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

File: types/underscore/index.d.ts [hunk 1]
```diff
@@ -3807,14 +3807,14 @@ declare module _ {
...
-        property(key: string): (object: Object) => any;
+        property(key: string | string[]): (object: Object) => any;
 
         /**
         * Returns a function that will itself return the value of a object key property.
         * @param key The object to get the property value from.
         * @return Function which accept a key property in `object` and returns its value.
         **/
-        propertyOf(object: Object): (key: string) => any;
+        propertyOf(object: Object): (key: string | string) => any;
```
Comment (weswigham) [inline types/underscore/index.d.ts]: `string | string` ...?

**Review by weswigham — CHANGES_REQUESTED** [review]

File: types/underscore/index.d.ts (hunk 1 above)
Comment (lleelleeoo) [reply types/underscore/index.d.ts]: Misstype. Fixed, thanks.

Comment (lleelleeoo) [reply types/underscore/index.d.ts]: @weswigham 
May You check the code one more time, please?

**Review by jgonggrijp — APPROVED** [review]
Code is correct. I could use this fix in my own code.

**Review by weswigham — APPROVED** [review]

**Pull request merged**